### DISCUSSION TITLE
How to set keymaps for debugging?

### DISCUSSION THREAD

**Discussion by SamPosh** [LazyVim/LazyVim#625] [Q&A]
How to set below keymaps for debugging?

-- Basic debugging keymaps, feel free to change to your liking!
    vim.keymap.set('n', '<F5>', dap.continue)
    vim.keymap.set('n', '<F1>', dap.step_into)
    vim.keymap.set('n', '<F2>', dap.step_over)
    vim.keymap.set('n', '<F3>', dap.step_out)
    vim.keymap.set('n', '<leader>b', dap.toggle_breakpoint)
    vim.keymap.set('n', '<leader>B', function()
      dap.set_breakpoint(vim.fn.input 'Breakpoint condition: ')
    end)

I am using this in kickstart.nvim. Now I am trying to enable dap support in lazy vim using this steps
https://www.lazyvim.org/plugins/extras/dap.nlua

But I don't know how to set these keys to enable debugging. Can some body help me?

**Comment by folke** [comment]
https://www.lazyvim.org/plugins/extras/dap.core

Go to the section for `nvim-dap` and click on `Full Spec`
